tests/orm/models/test_models.py

```diff
@@ -138,6 +143,16 @@ def required_arguments(request, default_user, aiida_localhost, tmp_path):
         return StructureData, {'cell': [[1, 0, 0], [0, 1, 0], [0, 0, 1]]}
     if request.param is RemoteData:
         return RemoteData, {'remote_path': '/some/path', 'computer': aiida_localhost}
+    if request.param is RemoteStashData:
+        return RemoteStashData, {'stash_mode': StashMode.COMPRESS_TAR}
+    if request.param is RemoteStashCompressedData:
+        return RemoteStashCompressedData, {
+            'stash_mode': StashMode.COMPRESS_TAR,
+            'target_basepath': '/some/path',
+            'source_list': ['/some/file'],
+            'dereference': True,
+            'computer': aiida_localhost,
+        }
```

Comment:
There's no such field as computer for stash options..
The fact that the test are passing is worrying! 

```suggestion
        return RemoteStashCompressedData, {
            'stash_mode': StashMode.COMPRESS_TAR.value,
            'target_basepath': '/some/path',
            'source_list': ['/some/file'],
            'dereference': True,
        }
```